### ISSUE TITLE
php-cs-fixer doesn't work with latest docker image version

### ISSUE THREAD

**Issue by bastien70** [jakzal/phpqa#398]
Hello, with latest version of phpqa, when we launch php-cs-fixer, we have the following error : 

```bash
/entrypoint.sh: 6: exec: php-cs-fixer: not found
task: Failed to run task "cs-fix": exit status 127
```

No errors with previous phpqa version (2 months ago), and no error with latest friendsofsymfony/php-cs-fixer version

**Comment by jakzal** [comment]
Have you switched to php 8.2 or use the latest? In the later case the switch was made for you.

php-cs-fixer isn’t supported on php 8.2 yet. See https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues?q=is%3Aissue+is%3Aopen+8.2+label%3Atopic%2FPHP8.2

**Comment by rvanlaak** [comment]
Support for PHP 8.2 "officially was enabled" ;-) last week with [v3.15.0](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/releases/tag/v3.15.0) through https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6825

see https://github.com/PHP-CS-Fixer/PHP-CS-Fixer#supported-php-versions

**Comment by jakzal** [comment]
Released as [v1.86.0](https://github.com/jakzal/phpqa/releases/tag/v1.86.0).
